### ISSUE TITLE
as tester I want to be able to add a file with test results in a github comment

### ISSUE THREAD

**Issue by DaanHoogland** [cloudops/upr#1]
when I upload test results for a PR with upr, I want to be able to add local log files in the git comment to describe issues and aid the PR submitter in resolving issues. This would be equivalent to dragging the files in the comment (as opposed to pasting the contents in the comment).

**Comment by swill** [comment]
If you use S3 or Swift you can do that now by specifying your credentials and where you want the files stored (and they can even be auto cleaned up after a period of time).

I will look into seeing if I can get them to upload to `https://cloud.githubusercontent.com/assets` like Github does natively...

**Comment by DaanHoogland** [comment]
[file.txt](https://github.com/cloudops/upr/files/207345/file.txt)
is saved as https://github.com/cloudops/upr/files/207345/file.txt (not in cloud.githubusercontent.com it seems)

**Comment by swill** [comment]
But can you upload a file like that without using the 'drag and drop' thing below?  Can you `curl` a file to somewhere and have it saved?  We can't count on being able to use the drag and drop when we are scripting.  We need to be able to upload while not using the Github UI.

**Comment by DaanHoogland** [comment]
Yes I know, I don't have the solution yet, btw there is a select box posibility as well in the bottom bar of the comment edit box
